tests/backup/backup_helper.go

```diff
@@ -5559,3 +5678,242 @@ func UpgradeKubevirt(versionToUpgrade string, workloadUpgrade bool) error {
 	}
 	return nil
 }
+
+// ChangeAdminNamespace changes admin namespace for Storage Cluster
+func ChangeAdminNamespace(namespace string) (*v1.StorageCluster, error) {
```

Comment:
Can we call this `ChangeStorkAdminNamespace` ?

Reply:
If we are not using `*v1.StorageCluster` returned by this function, we can just `error`